### PR TITLE
DOCSP-40654-sample-aggregation-operator

### PR DESCRIPTION
**Pull request by gmiller-mdb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-40654>

### Staging Links

<li><a href=https://deploy-preview-360--mongodb-docs-csharp.netlify.app/fundamentals/linq>fundamentals/linq</a></li>

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Review by mongoKart — CHANGES_REQUESTED** [review]
nice work! a few small things that i'm happy to help on 👍

File: source/fundamentals/linq.txt [hunk 1]
```diff
@@ -249,6 +249,34 @@ The result of the preceding example contains the following documents:
...
     { "name" : "Forlinis Restaurant", "cuisine" : "Italian" }
 
+$sample
+~~~~~~~
+
+The ``$sample`` aggregation stage returns a random sample of documents from a 
+collection. The following example shows how to generate a ``$sample`` stage using
+LINQ: 
+
+.. code-block:: csharp
```
[the remark this answers is not in the document]
Comment (gmiller-mdb) [reply source/fundamentals/linq.txt:259]: I've done a couple of tests and I unfortunately do not think the work-around is very feasible- there is issues with the string conversion that requires me adding a bunch of extra methods that I don't think are very efficient.

**Review by mongoKart — APPROVED** [review]
LGTM. thanks for putting in the time and effort to investigate LINQ

File: source/fundamentals/linq.txt [hunk 2]
```diff
@@ -249,6 +249,33 @@ The result of the preceding example contains the following documents:
...
+The result of the preceding example contains the following documents:
+
+.. code-block:: json
+
+   // Results Truncated
+
+   { "title" : "State of Play", "languages" : ["English"]}
+   { "title" : "Bhopal: A Prayer for Rain", "languages" : ["English", "Hindi"] }
+   { "title" : "Dead Presidents", "languages" : ["English"]}
+   { "title" : "The Rains Came", "languages" : ["English"] }
```
Comment (adelinowona) [inline source/fundamentals/linq.txt:276]: The results from the code samples in this page seem to be using the `restaurants` collection from the `sample_restaurants` database. However, the results here seem to be from the `movies` collection in the `sample_mflix` database. You should update the results here.

**Review by adelinowona — CHANGES_REQUESTED** [review]

**Review by adelinowona — APPROVED** [review]
LGTM

**Pull request merged**